### DISCUSSION TITLE
Tailwind CSS IntelliSense in cva in Neovim

### DISCUSSION THREAD

**Discussion by magoz** [joe-bell/cva#82] [Q&A] [answered]
As the [docs point out](https://github.com/joe-bell/cva#tailwind-css-intellisense), it's possible to get IntelliSense inside `cva`, adding the following to `settings.json`:

```
{
  "tailwindCSS.experimental.classRegex": [
    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
  ]
}
```

I'm using the [Tailwind LSP](https://github.com/tailwindlabs/tailwindcss-intellisense) in Neovim, and it works really well.  
Does anybody know if there is a Neovim equivalent to VScode's `settings.json` where I can define those settings per project?

Thanks!

**Comment by joe-bell** [comment, marked as answer]
[Maybe this?](https://github.com/joe-bell/cva/pull/53/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

**Comment by magoz** [comment]
Thank you, that works!

It might still be a good idea to add it to the docs.
I can report back if it needs to be updated, but it's unlikely since it uses the [tailwind LSP APIs](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#tailwindcss) (and in that case the VScode snippet would have to change too).
